### DISCUSSION TITLE
'Break' sequence

### DISCUSSION THREAD

**Discussion by helmuthk** [Eugeny/tabby#8548] [Q&A]
Hi,
I luurve Tabby! Been using it for 2 years already, but I'm in Routing, Switching and Security so I cannot issue a break to terminate any of the Cisco commands like traceroute, ping etc... which is frustrating as heck. If I start a traceroute that fails, I have to wait for all 30 hops to finish before being able to type in another command.
This takes a long time(30 seconds) as any person familiar with Cisco will tell you. This is unacceptable in a production environment when it's time for troubleshooting and maintenance, since time is of the essence.

I do not see the 'Break' sequence in the options/settings menu, so I wanted to know if we can add this feature in the next release, please?
This is a pretty basic feature and all(to my limited knowledge) terminal emulators have it.

Thank you,
Helmuth Kisting

**Comment by adgrant93** [comment]
FYI, on Mac you can use Ctrl+6 x. Maybe give that a try on on your OS and see if that allows you to break.

**Comment by redrob** [comment]
Any update on this one? Running Windows here, the break sequences I have tested do not seem to work at all either.

**Reply by adgrant93** [reply]
Try Ctrl+6 x. You will need to remap Ctrl I think.
